runelite-client/src/main/java/net/runelite/client/plugins/fairyring/FairyRingPlugin.java

```diff
@@ -27,17 +29,34 @@
 package net.runelite.client.plugins.fairyring;
 
 import com.google.common.eventbus.Subscribe;
+import com.google.inject.Provides;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
 import javax.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
 import net.runelite.api.Client;
+import net.runelite.api.ScriptID;
+import net.runelite.api.SpriteID;
 import net.runelite.api.Varbits;
+import net.runelite.api.WidgetType;
+import net.runelite.api.events.GameTick;
+import net.runelite.api.events.MenuOptionClicked;
 import net.runelite.api.events.VarbitChanged;
 import net.runelite.api.events.WidgetLoaded;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetID;
 import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.game.ChatboxInputManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.util.Text;
 
+@Slf4j
 @PluginDescriptor(
 	name = "Fairy Ring Helper"
```

Comment:
Probably rename this to Fairy Rings too. To follow configuration (and i see little point to show "Helper" when all plugins are helpers)